src/app/auth/auth-registration/auth-registration.component.html

```diff
@@ -0,0 +1,37 @@
+<ion-content color="light" text-center>
+  <h2>Registration</h2>
+  <div class="div-logo registration">
+    <img src="/assets/logo.svg" width="60%">
+  </div>
+  <div class="div-after-logo">
+    <ng-container>
+      <h4>Your Login Email Address</h4>
+      <p>{{user.email}}</p>
+    </ng-container>
+    <!-- <ng-container>
+        <h4>Your Mobile Number</h4>
+        <p>{{user.contact}}</p>
+    </ng-container> -->
+    <form (submit)="register()">
+      <ion-list margin>
+        <ion-item>
+          <ion-input name="passowrd" [(ngModel)]="password" (focus)="removeErrorMessages()" [ngModelOptions]="{standalone: true}" required type="password" placeholder="password"></ion-input>
+        </ion-item>
+        <ion-item>
+          <ion-input name="rePassword" [(ngModel)]="rePassword" (focus)="removeErrorMessages()" [ngModelOptions]="{standalone: true}" required type="password" placeholder="Conform Password"></ion-input>
+        </ion-item>
+        <ion-item lines="none">
+          <ion-checkbox color="primary" slot="start" (click)="validateRegistration('agree')" [(ngModel)]="isAgreed" [ngModelOptions]="{standalone: true}"></ion-checkbox>
```

Comment:
Don't need to do validation here, as mentioned above, use the Angular built in validator